api/turing/middleware/openapi_validation.go

```diff
@@ -105,3 +106,16 @@ func (openapi *OpenAPIValidation) Middleware(next http.Handler) http.Handler {
 	}
 	return handlerFunc
 }
+
+func jsonError(w http.ResponseWriter, msg string, status int) {
```

Comment:
This function is defined here since the file in which it was previously defined (`api/turing/middleware/authorization.go`) has been deleted.